apps/client/src/components/Questions.js

```diff
@@ -86,21 +91,25 @@ const Questions = ({
         },
       ]);
     }
+    if (questionIndex === 0) {
```

Comment:
Can this not be `else`?

Reply:
This can't be else because the last question on the stack isn't necessarily the question you are editing atm